src/NuGet.Core/NuGet.Commands/RestoreCommand/ProjectRestoreRequest.cs

```diff
@@ -22,7 +22,14 @@ internal class ProjectRestoreRequest
             ExistingLockFile = existingLockFile;
             MaxDegreeOfConcurrency = request.MaxDegreeOfConcurrency;
             Project = packageSpec;
-            PackageExtractionContext = request.PackageExtractionContext;
+            PackageExtractionContext = new PackageExtractionContext(
```

Comment:
If you aren't using `PackageExtractionContext` from `RestoreRequest` then remove that attribute from there since it's a required param to build restore request which is not even going to be used any where

Reply:
https://github.com/NuGet/NuGet.Client/pull/2497/files#diff-101420c352b3f5357ea7134fe678d121R25 restore request now doesn't have a `PackageExtractionContext`